packages/stripe_web/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 6.2.1
```

Comment:
Can you undo the versioning to 6.2.1 for now. I think with the new sync we probably go for a 6.3.0

Reply:
So should I'll remove this entirely then or change it to 6.3.0 ?

Reply:
I changed it to 6.3.0 and in the pubspec too.